tests/puppeteer/campaigns/sanity/catalogFO/02_checkProduct.js

```diff
@@ -0,0 +1,31 @@
+// Using chai
+const {expect} = require('chai');
+
+// Importing pages
+const HomePage = require('../../../pages/FO/home');
+const ProductPage = require('../../../pages/FO/product');
+const ProductData = require('../../data/FO/product');
+
+let page;
+let homePage;
+let productPage;
+
+// creating pages objects in a function
+const init = async () => {
+  page = await global.browser.newPage();
+  homePage = await (new HomePage(page));
+  productPage = await (new ProductPage(page));
+};
+
+/*
+  Open the FO home page
+  Check the first product page
+ */
+global.scenario('Check the Product page', () => {
+  test('should open the shop page', async () => {
```

Comment:
Same here, is it possible to do otherwise, usually it's considered bad practice, even in javascript, or is it considered a normal practice in puppeteer ?

Reply:
same here https://github.com/PrestaShop/PrestaShop/pull/15389#discussion_r320727041